math/maxima/Portfile

```diff
@@ -22,27 +22,50 @@ long_description \
         can plot functions and data in two and three dimensions.
 
 homepage        http://maxima.sourceforge.net/
-master_sites    sourceforge
 
-checksums       md5     3bccafb512ff1054be700c61601088af \
-                sha256  f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3 \
-                rmd160  3c4b153554c9b9cb3442e030d503fbe975a614fc
+variant devel description {Get a development version with git.} {}
+
+if {[variant_exists devel] && [variant_isset devel]} {
+    # git describe --tags : branch-5_41-base-415-g7e71fc4
+    version     5.41post
+    fetch.type  git
+    git.url     https://git.code.sf.net/p/maxima/code
+    git.branch  7e71fc4b6a95cab6418227754f01fc44ac845da1
+
+    post-fetch {
+        system -W ${worksrcpath} "git describe --tags"
+        system -W ${worksrcpath} "./bootstrap"
+    }
+    
+    livecheck.type  none
+} else {
+    # get the source tarball from sourceforge.
+    master_sites    sourceforge
+
+    checksums   md5     972c51384d7895c88d78eb045c6aedb2 \
```

Comment:
We no longer use md5 checksums in our portfiles, but we do now add a `size` parameter.

Reply:
I see. I will remove the md5 checksums line and add a size parameter.